.github/workflows/run-all-tests.yml

```diff
@@ -29,8 +29,11 @@ jobs:
           CI: true
         run: |
           export VAULT_TOKEN=$( vault write -field=token auth/approle/login role_id=$VAULT_ROLE_ID secret_id=$VAULT_SECRET_ID )
+          DOCKER_IMAGE_NAME="gcr.io/broad-singlecellportal-staging/single-cell-portal"
```

Comment:
Blocker: see note beginning "please add code comments".  (That said, this is a case where using "staging" is a bit more intuitive.)

Reply:
Perhaps:

`# "broad-singlecellportal-staging" GCR repository is also used in production.`